Modules/StudyProgramme/classes/class.ilStudyProgrammeChangeExpireDateGUI.php

```diff
@@ -39,7 +38,7 @@ public function __construct(
         ilObjUser $user,
         Factory $input_factory,
         Renderer $renderer,
-        ServerRequest $request,
+        Psr\Http\Message\ServerRequestInterface $request,
```

Comment:
Can we get rid of this completely in favour of `ILIAS\HTTP\Wrapper\RequestWrapper`?

Reply:
Not yet, the 'withRequest' method typehints on that type.